### ISSUE TITLE
OKCoin Error with credentials

### ISSUE THREAD

**Issue by frrrizzz** [butor/blackbird#413]
Hi,
let me first of all thank the author for sharing this project. 
Now the problem: is anyone else facing issues with OKCoin? 
When trying to retrieve the accuount balance I am facing an "Error with the credentials". I triple-checked the API key and secret, made sure that the api is activated and even deactivated 2FA, passphrase etc. in order to avoid any disturbances by additional security measures - but still I am receiving this error. 
I have been verified as Lvl1 user and I am still in the process of validation as Lvl2 user on OKCoin. Can this be the reason? The point is, that I was already able to deposit and trade BTC successfully, so didn't expect any issues to arise using the API.

Thank you

**Comment by Iheff** [comment]
I would try just connecting to their API with your credentials away from
blackbird.. to reduce the test case.
If that works, then its worth looking into blackbird however if you cant
connect via a basic standalone integration then the problem is on their
side, or with the keys you have (might be worth requesting a new set)

**Comment by frrrizzz** [comment]
Hi Iheff,

I've thought about doing so, but to be honest, I first need to look into the REST API doc and figure out, how such test scenario could look like. Simply calling the URL from my browser is too simple, isn't it?

**Comment by Iheff** [comment]
you might be able to use POSTMAN or something like that, but to be honest
there is probably like a wrapper in node or front-end JS that you can get
going quite fast, search github for OKCOIN API try this one:
https://github.com/etiennetatur/OKCoin-API

Best of luck.

**Comment by frrrizzz** [comment]
Thank you for your quick reply. I've tested it with the C# Code OKCoin provides on their [homepage](https://www.okcoin.com/rest_sample.html) and it seems like even this one fails with "Server not found" error. The code example is quite straight forward but still I am not sure whether I did something wrong. If anyone is using OKCoin successfully, this would be a welcome evidence, that I am not doing things properly. 
Kind regards

**Comment by CCKRK** [comment]
You've queried other exchanges with success and only Okcoin is giving you an error?

**Comment by frrrizzz** [comment]
Yes. I am currently running blackbird with a total of 4 exchanges and only OKCoin is giving an error. I have a OKCoin.com (not OKCoin.cn) account which works when accessed via browser. 
Kind regards

**Comment by frrrizzz** [comment]
Any News on this one? Is anyone successfully using OKCoin at the Moment? I still get the "Error with the Credentials" error.

**Comment by shermhead** [comment]
I have the same issue. Probably because I am not verified.

**Comment by igladun** [comment]
I have the same issue, and I am also not verified.

**Comment by wwarlock** [comment]
Looks like OKCoin API was changed.
Now they are required to use `/api/v1/` and without `.do`-postfix urls.
For example
`https://www.okcoin.com/api/v1/ticker`
Instead of
`https://www.okcoin.com/api/ticker.do?ok=1`
